streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBStoreTest.java

```diff
@@ -220,6 +221,50 @@ public void shouldRestoreAll() throws Exception {
             "c");
     }
 
+    @Test
+    public void shouldPutOnlyIfAbsentValue() throws Exception {
```

Comment:
Could we put this test to `AbstractKeyValueStoreTest` so that all types of kv stores will test it? Ditto below.

Reply:
`RocksDBStoreTest` does not extend `AbstractKeyValueStoreTest`, so added the method to `AbstractKeyValueStoreTest`.  Maybe a follow up PR to update `RocksDBStoreTest`?